### ISSUE TITLE
Add Multiple or Custom Enzymes

### ISSUE THREAD

**Issue by jackrogan** [bigbio/quantms#273]
### Description of feature

Is it possible to run a search with multiple enzymes, or to add a custom enzyme (in this case with the appropriate cleaving locations expected for a 2-enzyme digestion?)

If it helps, the digestions I'm interested in running are Trypsin + Asp-N and Trypsin + Glu-C.

Thanks!

**Comment by ypriverol** [comment]
@jackrogan this is not supported. Yet, we will give priority for next release. Can you let me know which dataset do you have in mind that can help us to test.

**Comment by jpfeuffer** [comment]
Yes, it is a bit complicated. In theory, we need to model any order of enzymes and add an option to allow for simultaneous application. While e.g. Comet supports this, many other search engines do not.
We could probably release a Comet-only version of this feature earlier than a general one.

**Comment by ypriverol** [comment]
I know already that when I'm using enzymes different that Trypsin, I need to enable only `Comet`, this is already happening because, examples Trypsin/P doesn't work properly with multi search engine. 

As you said similar approach can be followed for multiple enzymes, only allow Comet when multiple enzymes is needed.

**Comment by jpfeuffer** [comment]
> I know already that when I'm using enzymes different that Trypsin, I need to enable only `Comet`, this is already happening because, examples Trypsin/P doesn't work properly with multi search engine. 
> 
> As you said similar approach can be followed for multiple enzymes, only allow Comet when multiple enzymes is needed. 

Trypsin/P should work. Please open another issue for this.
